pkg/components/util/install.go

```diff
@@ -140,3 +135,28 @@ func InstallAsRelease(name string, c components.Component, kubeconfig string) er
 
 	return nil
 }
+
+// HelmActionConfig creates initialized Helm action configuration.
+func HelmActionConfig(ns string, kubeconfig string) (*action.Configuration, error) {
+	actionConfig := &action.Configuration{}
+
+	// TODO: Add some logging implementation? We currently just pass an empty function for logging.
+	if err := actionConfig.Init(kube.GetConfig(kubeconfig, "", ns), ns, "secret", func(format string, v ...interface{}) {}); err != nil {
+		return nil, fmt.Errorf("failed initializing helm: %v", err)
+	}
+
+	return actionConfig, nil
+}
+
+// ReleaseExists checks if given Helm release exists.
+func ReleaseExists(actionConfig action.Configuration, name string) (bool, error) {
+	histClient := action.NewHistory(&actionConfig)
+	histClient.Max = 1
+
+	_, err := histClient.Run(name)
+	if err != nil && err != driver.ErrReleaseNotFound {
+		return false, fmt.Errorf("failed checking for chart history: %v", err)
```

Comment:
Should we use `%w` here?
```suggestion
		return false, fmt.Errorf("failed checking for chart history: %w", err)
```

Reply:
Yep.